test/lint/lint-submodule.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2020 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
```

Comment:
It will be nice having here a description (smth like "Check for git submodules.") following an empty line (see other scripts in this subdir as en example).